app/bundles/UserBundle/MauticUserBundle.php

```diff
@@ -13,6 +18,11 @@ public function build(ContainerBuilder $container): void
         parent::build($container);
 
         $extension = $container->getExtension('security');
-        $extension->addSecurityListenerFactory(new PluginFactory());
+        \assert($extension instanceof SecurityExtension);
+        $extension->addAuthenticatorFactory(new PluginFactory());
+        $extension->addAuthenticatorFactory(new MauticSsoFactory());
+
+        $container->addCompilerPass(new OAuthReplacePass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, 0);
+        $container->addCompilerPass(new SsoAuthenticatorPass(), PassConfig::TYPE_BEFORE_OPTIMIZATION, 0);
```

Comment:
```suggestion
        $container->addCompilerPass(new OAuthReplacePass());
        $container->addCompilerPass(new SsoAuthenticatorPass());
```
Let's remove unnecessary arguments.

Reply:
This is made to conform the same code as in other core bundles, e.g. https://github.com/mautic/mautic/blob/5.1.1/app/bundles/CoreBundle/MauticCoreBundle.php#L14